<launch>

    <node name="Auto_Interface" pkg="mavric" type="Autonomous_P.py" respawn="true">
    </node>

    <node name="Auto_System" pkg="mavric" type="complex_main.py" respawn="true">
        <remap from="GPS_Fix" to="/HW/GPS_Fix" />
        <remap from="GPS" to="/HW/GPS_Data" />
        <remap from="SensorCalibrations" to="/HW/IMU/SensorCalibrations" />
        <remap from="FusedAngle" to="/HW/IMU/FusedAngle" />
        <remap from="Drive_Train" to="/Drive/Drive_Command" />
        <remap from="Steer_Train" to="/Drive/Steer_Command" />
    </node>

    <node name="GPS_Sim" pkg="mavric" type="GPS_Sim.py" respawn="true">
        <remap from="FusedAngle" to="/HW/IMU/FusedAngle" />
        <remap from="Drive_Train" to="/Drive/Drive_Command" />
        <remap from="GPS_Data" to="/HW/GPS_Data" />
    </node>
    
</launch>